File: Iniciante/URI 1046.lua
line = {}
n = 0
for i in string.gmatch(io.read("*l"), "%S+") do
   n = n + 1
   line[n] = i
end
hInicial = tonumber(line[1])
hFinal = tonumber(line[2])

if (hInicial > hFinal) then
    print('O JOGO DUROU ' .. (24 - (hInicial - hFinal)) .. ' HORA(S)')
elseif (hFinal > hInicial) then
    print('O JOGO DUROU ' .. (hFinal - hInicial) .. ' HORA(S)')
else
    print('O JOGO DUROU 24 HORA(S)')
end

File: Iniciante/URI 1046.clj
(require '[clojure.string :as str])

(def line (str/split (read-line) #" "))
(def hInicial (Integer/parseInt (get line 0)))
(def hFinal (Integer/parseInt (get line 1)))

(cond
    (> hInicial hFinal) 
        (printf "O JOGO DUROU %d HORA(S)\n" (- 24 (- hInicial hFinal)))
    
    (> hFinal hInicial) 
        (printf "O JOGO DUROU %d HORA(S)\n" (- hFinal hInicial))
    
    :else 
        (printf "O JOGO DUROU 24 HORA(S)\n")   
)